app/services/csv_school_moves.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+class CSVSchoolMoves
+  include ServicePattern
```

Comment:
The `ServicePattern` is something we did at DfE https://github.com/DFE-Digital/register-trainee-teachers along with a `services/` directory. We don't have to follow it, but I thought it's worth putting up for show and get some feedback.